### PR TITLE
feat!: remove all crates other than `acvm-backend-barretenberg` and remove workspace

### PR DESCRIPTION
**Pull request by TomAFrench**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,65 @@
-[workspace]
-
-members = ["common", "acvm_backend_barretenberg", "aztec_backend_wasm"]
-
-[workspace.package]
+[package]
+name = "acvm-backend-barretenberg"
+version = "0.1.0"
```
Comment (phated) [inline Cargo.toml:3]:
```suggestion
version = "0.0.0"
```
So we can make the first release with RP and it'll be 0.1.0

**Review by phated — APPROVED** [review]
Just want to drop that version to 0.0.0, but otherwise looks good.

**Comment by phated** [comment]
Ah, and the wasm workflow needs to be deleted

**Comment by TomAFrench** [comment]
I'm also getting some issues in flake locally. I'm just checking if CI experiences the same atm.

**Comment by TomAFrench** [comment]
Removing the todo as we should rename the repo after this PR.

**Comment by phated** [comment]
I just removed the build wasm required check to get this queued

**Pull request merged**